interceptor/config/tracing.go

```diff
@@ -0,0 +1,21 @@
+package config
+
+import (
+	"github.com/kelseyhightower/envconfig"
+)
+
+// Tracing is the configuration for configuring tracing through the interceptor.
+type Tracing struct {
+	// States whether tracing should be enabled, False by default
+	Enabled bool `envconfig:"KEDA_HTTP_TRACING_ENABLED" default:"false"`
+	// Sets what tracing export to use, must be one of: console,otlphttp
+	Exporter string `envconfig:"KEDA_HTTP_TRACE_EXPORTER" default:"console"`
+}
```

Comment:
[We are going to rename](https://github.com/kedacore/http-add-on/pull/1039/files) other envs to be aligned with the spec, so we have to change these 2 too

What about 
```suggestion
type Tracing struct {
	// States whether tracing should be enabled, False by default
	Enabled bool `envconfig:"OTEL_EXPORTER_OTLP_TRACES_ENABLED" default:"false"`
	// Sets what tracing export to use, must be one of: console,otlphttp
	Exporter string `envconfig:"OTEL_EXPORTER_OTLP_PROTOCOL" default:"console"`
}
```

Reply:
refactored as per comment